test/ACSets.jl

```diff
@@ -73,7 +73,18 @@ for dds_maker in dds_makers
   view(dds,:Φ) isa ColumnView
 
   # Deletion.
+  @test_throws ErrorException undefined_subparts(dds, :X)
+  if dds.parts[dom(acset_schema(dds),:Φ)] isa DenseParts
```

Comment:
This might be a good time to implement a function which gets the parts type from an ACSet, which is known at the type level. E.g. `parts_type(::ACSet{PT}) where PT = PT`. 

Then this test could become `if parts_type(dds) <: DenseParts `

Reply:
Great suggestion.

Reply:
unfortunately now this function isn't being tested - maybe you can throw a line in somewhere like `@test typeof(dds.parts[:X]) isa parts_type(dds)`

Reply:
whoops.